BridgeCareApp/BridgeCare/Services/SummaryReport/CostBudgetsWorkSummary.cs

```diff
@@ -0,0 +1,222 @@
+﻿using BridgeCare.Models;
+using OfficeOpenXml;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Web;
+
+namespace BridgeCare.Services
+{
+    public static class CostBudgetsWorkSummary
```

Comment:
For all of these report classes, I don't think they can be static. They each contain state information that may change from user to user that is being written out to the report correct? What happens if multiple users try to run the same report at the same time?

Reply:
Yes, that might create problem. Changing those to regular classes.

Reply:
Updated.